web/src/popover_menus.js

```diff
@@ -328,6 +335,43 @@ export function do_schedule_message(send_at_time) {
     compose.finish(true);
 }
 
+export function version_display_string() {
+    const version = page_params.zulip_version;
+    const is_fork = page_params.zulip_merge_base && page_params.zulip_merge_base !== version;
+
+    if (page_params.zulip_version.endsWith("-dev+git")) {
+        // The development environment uses this version string format.
+        return $t({defaultMessage: "Zulip Server dev environment"});
+    }
+
+    if (is_fork) {
+        // For forks, we want to describe the Zulip version this was
+        // forked from, and that it was modified.
+        const display_version = page_params.zulip_merge_base
+            .replace(/\+git.*/, "")
+            .replace(/-dev.*/, "-dev");
+        return $t({defaultMessage: "Zulip Server {display_version} (modified)"}, {display_version});
+    }
+
+    // The below cases are all for official versions; either a
+    // release, or Git commit from one of Zulip's official branches.
+
+    if (version.includes("+git")) {
+        // A version from a Zulip official maintenance branch such as 5.x.
+        const display_version = version.replace(/\+git.*/, "");
+        return $t({defaultMessage: "Zulip Server {display_version} (patched)"}, {display_version});
+    }
+
+    const display_version = version.replace(/\+git.*/, "").replace(/-dev.*/, "-dev");
+    return $t({defaultMessage: "Zulip Server {display_version}"}, {display_version});
+}
+
+export function open_gear_menu() {
+    $("#settings-dropdown").trigger("click");
+    // there are invisible li tabs, which should not be clicked.
+    $("#gear-menu").find("li:not(.invisible) a").eq(0).trigger("focus");
```

Comment:
dont think this is valid  anymore. doesn't do anything.
`.invisible` class is also not present